invoicing_app/management/commands/generate_tax_receipts.py

```diff
@@ -166,24 +167,15 @@ def handle(self, **options):
 
         if options['event_id']:
             self.event_id = options['event_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`id` = ' + str(self.event_id)
-            )
+            self.conditional_mask = 'AND `Events`.`id` = ' + str(self.event_id)
```

Comment:
When working with strings its recommended to use `format` rather than using the `+` operator, in this case it will be like this: `'AND `Events`.`id` = {}'.format(self.event_id)`, and in Python3 you would use something like `f'AND `Events`.`id` = {self.event_id}` .

Reply:
Thanks! Done!